target/linux/pistachio/marduk/profiles/marduk_ca8210.mk

```diff
@@ -28,7 +28,8 @@ define Profile/marduk_ca8210
     PACKAGES:=kmod-i2c kmod-cascoda kmod-sound-pistachio-soc \
               wpan-tools tcpdump uhttpd uboot-envtools \
               alsa-lib alsa-utils alsa-utils-tests i2c-tools \
-              iw hostapd wpa-supplicant kmod-uccp420wlan kmod-cfg80211
+              iw hostapd wpa-supplicant kmod-uccp420wlan kmod-cfg80211 \
+              kmod-leds-gpio kmod-ledtrig-heartbeat kmod-ledtrig-timer
```

Comment:
it seems that target/linux/generic/config-4.1 already has 
CONFIG_LEDS_TRIGGERS=y
CONFIG_LEDS_TRIGGER_DEFAULT_ON=y
CONFIG_LEDS_TRIGGER_NETDEV=y
CONFIG_LEDS_TRIGGER_TIMER=y
so we won't need kmod-ledtrig-heartbeat or kmod-ledtrig-timer explicitly in the marduk profiles.